basket/news/views.py

```diff
@@ -939,3 +955,36 @@ def update_user_task(request, api_call_type, data=None, optin=False, sync=False)
         return HttpResponseJSON({
             'status': 'ok',
         })
+
+
+@require_POST
+@csrf_exempt
+def subhub_post(request):
+    if not has_valid_api_key(request):
+        return HttpResponseJSON({
+            'status': 'error',
+            'desc': 'requires a valid API-key',
+            'code': errors.BASKET_AUTH_ERROR,
+        }, 401)
+
+    try:
+        data = json.loads(request.body)
+    except ValueError as _:
```

Comment:
If you don't need the exception object just leave the `as _` off.
```suggestion
    except ValueError:
```